server/src/main/java/org/elasticsearch/index/mapper/SeqNoFieldMapper.java

```diff
@@ -69,26 +69,29 @@
         public final Field seqNo;
         public final Field seqNoDocValue;
         public final Field primaryTerm;
+        public final Field tombstoneField;
```

Comment:
isn't this a softDeleteField?

Reply:
No, this is not a softDeletes field. This field is used by a delete-op and an noop  only.

Reply:
doh. I got confused. Sorry.